pyproject.toml

```diff
@@ -17,8 +17,6 @@ classifiers = [
 python = ">=3.8,<4.0"
 confluent-kafka = ">= 1.9.2"
 aiokafka = ">=0.10,<0.12"
-pytest-cov = ">=4.1,<6.0"
-pytest-asyncio = "^0.23.5"
```

Comment:
Please could these be _moved_ to the dev dependencies section rather than dropped? I realise that `pytest-cov` is already mentioned there, though with a slightly different version range. `pytest-asyncio` isn't mentioned there at all, which I expect will break CI.